src/spectral/rulesets/.defaultsForSpectral.yaml

```
@@ -244,3 +244,14 @@ rules:
       - $.servers[*][variables][*][default]
     then:
       function: truthy
+  # ensure HEAD request does not have response body
+  head-request-should-not-have-response-body:
```

This rule is tough to name because we want it to be short yet descriptive. Those two qualities seem mutually exclusive here!

I think we should align with our other response-specific rules, `response-error-response-schema` and `response-example-provided`. I have a suggestion below. Feel free to suggest another name if you think of it!

```suggestion
  response-head-response-with-content:
```